### PR TITLE
Additional Auth0 documentation

### PR DESCRIPTION
**Pull request by brenetic**
A bit more documentation around the different connections

### CODE REVIEW

File: guides/auth0.md [hunk 1]
```diff
@@ -11,10 +11,16 @@ There are two connections configured currently, Google and Microsoft. The former
 
 Currently the only user data we keep is their name and email address, both of which are encrypted before being saved in the DB.
 
-The Universal Login screen can be configured directly in the moj-forms tenant dashboard. It is possible to create custom HTML and JS.
+The Universal Login has been configured within the Auth0 dashboard. Any changes should be done there directly. The username and password field are required only in editor-test as they are used by the fb-acceptance-tests user.
 
 There are two editor apps, one for Live and one for Test. Aside from the obvious differences between callback URLs, the Test app also has the username and password fields active in order for the acceptance tests to be able to log in.
 
+The tenant environment is usually set to "development" and needs to have a manual request through the Auth0 service desk to set it to "production". The best situation is to have separate tenants per environment however it so long to get the connections set up that to go back and request IDs and private keys just is not worth the time and effort.
```
Comment (njseeto) [inline guides/auth0.md]: just missing the word 'took'  in '...tenants per environment however it so long...'

File: guides/auth0.md [hunk 2]
```diff
@@ -11,10 +11,16 @@ There are two connections configured currently, Google and Microsoft. The former
...
+
+The Azure client secret will expire in 2 years. At this time it will need to be regenerated by the team who is responsible for the Microsoft Azure Active Directory system within the MoJ.
```
Comment (njseeto) [inline guides/auth0.md]: Might be worth adding when the 2 years will be, eg: April 2023.

**Review by njseeto — COMMENTED** [review]
Just a few small comments, otherwise it looks good 🚀

File: guides/auth0.md (hunk 2 above)
Comment (tomas-stefano) [reply guides/auth0.md]: Is it worth stating the team that is responsible for Microsoft at the moment?

**Comment by brenetic** [comment]
@tomas-stefano @njseeto thanks for the comments. Have updated

**Review by tomas-stefano — APPROVED** [review]
🚀

**Pull request merged**